src/io/GeoJSONReader.cpp

```diff
@@ -98,7 +98,14 @@ GeoJSONFeature GeoJSONReader::readFeature(const geos_nlohmann::json& j) const
 {
     const auto& geometryJson = j.at("geometry");
     const auto& properties = j.at("properties");
-    return GeoJSONFeature{readGeometry(geometryJson), readProperties(properties)};
+
+    std::string id = "";
+    if (j.contains("id") && !j.at("id").is_null()) {
+        if (j.at("id").is_string()) id = j.at("id").get<std::string>();
+        if (j.at("id").is_number()) id = std::to_string(j.at("id").get<int>());
```

Comment:
does get<int>() works fine on floating-point numbers ?

Reply:
It works fine but I think it is not correct. According to the geojson standard, all number is valid. So we have to support float point. I will upload a patch soon.